tests/support/xenial.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:xenial
+
+ENV container docker
+
+VOLUME [ "/sys/fs/cgroup" ]
+
+RUN echo 'root:root' | chpasswd
+
+RUN apt-get -y update
+RUN apt-get -y install ca-certificates=20160104ubuntu1 \
+                       git='1:2.7.4-0ubuntu1' \
+                       gcc='4:5.3.1-1ubuntu1' \
+                       openssh-client='1:7.2p2-4ubuntu2.1' \
+                       openssl=1.0.2g-1ubuntu4.5 \
+                       python-pip=8.1.1-2ubuntu0.2 \
+                       python-dev=2.7.11-1 \
+                       libffi-dev=3.2.1-4 \
+                       libssl-dev=1.0.2g-1ubuntu4.5
```

Comment:
IMHO, without version pinning, we'll get into the "works on my machine" discussions again.

what is a Dockerfile, without the statically defined things? For example, a few months from now, Jack will take this file and he'll build it, but it could be that he'll be installing a newer version of pip that doesn't know of our supported Ansible version.

I suggest we let it there and when it'll break because some versions were removed from the repos (very rare occasion if you ask me), we'll revisit this.